run_das.sh

```diff
@@ -0,0 +1,14 @@
+source venv/bin/activate
+
+# Run Spark Job
+PYSPARK_PYTHON=./VENV/venv/bin/python3 ../spark/spark-2.4.0-bin-hadoop2.7/bin/spark-submit \
+--master yarn \
+--conf spark.pyspark.virtualenv.enabled=true \
+--conf spark.pyspark.virtualenv.type=native \
+--conf spark.pyspark.virtualenv.requirements=requirements.txt \
+--conf spark.pyspark.virtualenv.bin.path=./VENV/venv/bin/virtualenv \
+--conf spark.yarn.appMasterEnv.PYSPARK_PYTHON=./VENV/venv/bin/python \
+--archives venv.zip#VENV \
+src/spark.py --es "$1" --f "$2"
```

Comment:
I think we need to pass input and output file, right?

Reply:
I didn't add an output path for the spark job since the output will be in HDFS and will be "hidden" from the "user".
The final output the user receives in the local storage will be renamed to the output file he has requested.